RELEASE.md

```diff
@@ -0,0 +1,74 @@
+# Versioning  
+Versions will follow the [semantic versioning](https://semver.org/) system.  
+The following guidelines will be used to determine when each version component will be updated:
+- **major**: backwards-breaking changes 
+- **minor**: functionality added in a backwards compatible manner
+- **patch**: backwards compatible bug fixes
+
+# Release Process
+### Pre-Release Tasks
+
+#### Start Release Ticket
+Move the JIRA ticket for the release to the "In Progress" state.
+Ensure that its fixVersion matches the version being released.
+
+#### Complete the Release in JIRA
+Go to the [SQL releases page](https://jira.mongodb.org/projects/SQL?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=unreleased), and ensure that all the tickets in the fixVersion to be released are closed.
+Ensure that all the tickets have the correct type. Take this opportunity to edit ticket titles if they can be made more descriptive.
+The ticket titles will be published in the changelog.
+
+If you are releasing a patch version but a ticket needs a minor bump, update the fixVersion to be a minor version bump.
+If you are releasing a patch or minor version but a ticket needs a major bump, stop the release process immediately.
+
+The only uncompleted ticket in the release should be the release ticket.
+If there are any remaining tickets that will not be included in this release, remove the fixVersion and assign them a new one if appropriate.
+
+Close the release on JIRA, adding the current date (you may need to ask the SQL project manager to do this).
+
+#### Ensure Evergreen Passing
+Ensure that the build you are releasing is passing the tests on the evergreen waterfall.
+
+### Release Tasks
+
+#### Ensure master up to date
+Ensure you have the `master` branch checked out, and that you have pulled the latest commit from `mongodb/mongo-tableau-connector`.
+
+#### Ensure plugin-version is correct
+Ensure that the `plugin-version` in [manifest.xml](https://github.com/mongodb/mongo-tableau-connector/blob/master/connector/manifest.xml) 
```

Comment:
Oh, I see.  I had the idea that we would manually update the version in the manifest.xml.  
How about for {placeholder} I put '0.0.0' so that the taco build will pass the xml validation.  And then if tag-triggered it will find/replace the value with the tag?  The non-tag-triggered will keep the placeholder '0.0.0'.

Reply:
Yes, sounds good to me. 0.0.0 seems very reasonable as a place holder 👍 
How uptight is Tableau about the version format, do you know?
For non tag triggered, to differentiate them, maybe we could see how to add the git version after the patch number? I'll create an investigation ticket for this since it does not apply to this JIRA itself, but it's good to keep track of it.

Reply:
I experimented earlier with setting it to something like '0.0.0-78b19e4'.  It doesn't work as the validation check seems to want only digits in the version.  It checks against this pattern for validation: 
`['(\\d+)((\\.)(\\d+))*((-)(\\d+))*']`

Reply:
ok, cool, it's nice you already checked. I'll add this as a comment on the related JIRA then. Maybe we can simply add the version as a suffix for the taco file name.